torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -937,6 +1062,9 @@ def _lower_to_native_backend(
     operator signature so they can be lowered with the same function
     """
     _lower_static_weighted_ref_module(model, qconfig_map)
+    if torch.backends.quantized.engine == 'onednn':
+        # Currently, this pass only works for onednn with conv2d_add and conv2d_add_relu fusion
+        _lower_static_weighted_ref_module_with_two_dq_inputs(model, qconfig_map)
```

Comment:
This rewrite function sounds general to me, not needed to add a special `onednn` backend check here?

Reply:
Thanks for comment, remove the checks here.